.github/workflows/version-management.yml

```
@@ -0,0 +1,222 @@
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Make scripts executable
+      run: chmod +x scripts/version-manager.sh
+
+    - name: Determine version bump type
+      id: bump_type
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:27: Double quote to prevent globbing and word splitting

(shellcheck)